test/JLSO.jl

```diff
@@ -7,10 +6,6 @@ using Distributions
 using Nullables  # Needed for loading BSON encoded ZonedDateTimes on 1.0
```

Comment:
Can you open an issue about this on TimeZones? Then I can keep track of these.

Reply:
just for info, TimeZones 0.8.4 contains "Remove Nullables" (https://github.com/JuliaTime/TimeZones.jl/compare/v0.8.4...master)

Reply:
When I test this package it looks like it's using TimeZones v0.8.5, so this is still an issue after that change I guess.

Reply:
@nickrobinson251 the next non-patch release of TimeZones will drop Nullables support. The issue is mostly so that I can track these indirect dependencies and make PRs to remove them once that release of TimeZones is out.